server/graphql/v1/CollectiveInterface.js

```diff
@@ -1776,8 +1770,8 @@ const CollectiveFields = () => {
         if (args.service) {
           paymentMethods = paymentMethods.filter(pm => pm.service === args.service);
         }
-        if (args.types) {
-          paymentMethods = paymentMethods.filter(pm => args.types.includes(pm.type));
+        if (args.type) {
+          paymentMethods = paymentMethods.filter(pm => args.type.includes(pm.type));
```

Comment:
Similar feedback here, it seems to duplicate the two line below

Reply:
Ops, corrected that one as well. Thanks for catching that. 🙂